src/swipe/dto/create-swipe.dto.ts

```diff
@@ -0,0 +1,13 @@
+import { IsString } from 'class-validator';
+import { UserSwipeType } from '../entities';
+
+export class SwipeUserDto {
+    @IsString()
+    target: string;
+
+    @IsString()
+    submittedBy: string;
+
+    @IsString()
+    status: UserSwipeType;
```

Comment:
This is critical for security. We don't want to allow user to specify any type of swipe data.

- submittedBy will always be current logged in user's id, we cannot pass that!

- target should be grabbed from path params, e.g. /swipe/:id

- status should be hidden from user: there should be two endpoints put and delete (just like in miro docs)

soooo this whole dto should go!